src/graphql/query/flssLoansQuery.graphql

```diff
@@ -0,0 +1,21 @@
+#import '../fragments/loanCardFields.graphql'
+
+query flssLoanData(
+  $filterObject: [FundraisingLoanSearchFilterInput!]
+  $imgDefaultSize: String = "w480h360"
+  $imgRetinaSize: String = "w960h720"
+) {
```

Comment:
Looks good so far. Please add in variable handling for the `limit` and `pageNumber` so we can query for a specific number of loans to show and handle paging too.

Reply:
cool. thanks for 👀

Reply:
so add
$limit: int = (is there a default value you like?)
$pageNumber: int (no particular default)?

Reply:
We generally don't set defaults for these as the api should be defaulting to 20 for limit and 0 for pageNumber. Offset is how we usually term what is "pageNumber" in this case.

If there are no defaults in flss we can use 20 + 0

Reply:
Thank you!

Reply:
Whoop! thank YOU! 🎉